src/core/Tooltip/TooltipContent/TooltipContent.tsx

```diff
@@ -51,7 +52,9 @@ class BaseTooltipContent extends Component<TooltipContentProps & InnerRef> {
       <HtmlDivWithRef
         {...passProps}
         forwardedRef={forwardedRef}
-        className={classNames(tooltipContentClassNames.content, className)}
+        className={classNames(tooltipContentClassNames.content, className, {
+          [tooltipContentClassNames.arrowHidden]: arrowOffsetPx < 1,
```

Comment:
Should this be arrowOffsetPx <= 0 in case of sub pixels, or does it matter in this context?